recipes/keggcharter/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "keggcharter" %}
+{% set version = "0.0.3" %}
+{% set sha256 = "3031f4aed298df068aa6fd3d9f70b10b4b6974eb6221abb3e4fcccec74488f98" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/iquasere/KEGGCharter/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: generic
+  number: 0
+  script: "mkdir -p $PREFIX/bin && cp *.py KEGGCharter_prokaryotic_maps.txt $PREFIX/bin"
```

Ideally, we'd have only executables in `${PREFIX}/bin`.
As a start, you could do something like
```suggestion
  script: |
    dir="${PREFIX}/share/KEGGCharter"
    mkdir -p "${dir}"
    cp *.py KEGGCharter_prokaryotic_maps.txt "${dir}/"
    mkdir -p "${PREFIX}/bin"
    chmod +x "${dir}/kegg_charter.py"
    ln -s "${dir}/kegg_charter.py" "${PREFIX}/bin/"
```
(Or you could turn this into a Python package and let `setuptools`/`pip` (or another build tool of your choice!) handle these kinds of things, of course ;).)

---

I have never used package managers, tho I feel is becoming necessary when submitting tools to bioconda... for now this approach of putting everything in bin was doing the trick, I didn't know you wanted only binaries in bin, but it makes sense xD I like this alteration, where could setuptools or pip help in it?